rules/call_peak.smk

```diff
@@ -1,24 +1,33 @@
+def get_genrich_replicates(wildcards):
+    assembly_ish, sample_condition = "-".join(wildcards.fname.split('-')[:-1]), wildcards.fname.split('-')[-1]
+    assembly = assembly_ish.split("/")[-1]
+
+    if sample_condition in samples.index or (sample_condition in treps.index and config.get("technical_replicates", "") == "keep"):
```

Comment:
can be simplified to 
`if sample_condition in treps.index:`

Reply:
Yep